includes/database/tables/class-order-adjustments.php

```diff
@@ -64,7 +65,8 @@ protected function set_schema() {
 		$this->schema = "id bigint(20) unsigned NOT NULL auto_increment,
 		object_id bigint(20) unsigned NOT NULL default '0',
 		object_type varchar(20) DEFAULT NULL,
-		type_id bigint(20) unsigned NOT NULL default '0',
+		type_id bigint(20) unsigned DEFAULT NULL,
+		type_key varchar(32) DEFAULT '',
```

Comment:
```suggestion
		type_key varchar(32) DEFAULT NULL,
```

Prefer a null default.

Reply:
Also I'm wondering if we should make it longer than `32` given that the old-style ID field is kind of a free for all.

Reply:
The `meta_key` in the `postmeta` table is set to 255.